proxy/http2/HTTP2.h

```diff
@@ -84,6 +84,12 @@ enum {
   HTTP2_STAT_SESSION_DIE_EOS,
   HTTP2_STAT_SESSION_DIE_ERROR,
   HTTP2_STAT_SESSION_DIE_HIGH_ERROR_RATE,
+  HTTP2_STAT_MAX_SETTINGS_PER_FRAME_EXCEEDED_RATE,
```

Comment:
The stats are not rates but counts, right? It may be just me, but I was confused a bit. Maybe we can omit "_RATE".

Reply:
hmmm right, they count the number of times that a specific rate limit has been exceeded, so we could drop the _RATE or reposition it.. like HTTP2_STAT_MAX_SETTINGS_PER_FRAME_RATE_EXCEEDED

Reply:
I'm also fine with the repositioning.

Reply:
I've dropped it after checking the naming, you were right :)